jwala-services/src/test/java/com/cerner/jwala/service/jvm/impl/JvmServiceImplTest.java

```diff
@@ -1006,7 +1009,7 @@ public JvmService getJvmService() {
             return new JvmServiceImpl(mockJvmPersistenceService, mockGroupPersistenceService, mockApplicationService,
                     mockMessagingTemplate, mockGroupStateNotificationService, mockResourceService, mockClientFactoryHelper,
                     "/topic/server-states", mockJvmControlService, mockBinaryDistributionService, mockBinaryDistributionLockManager,
-                    mockHistoryFacadeService, new FileUtility());
+                    mockHistoryFacadeService, mockWebServerPersistenceService, new FileUtility());
         }
 
     }
```

Comment:
You made this class compile, but where is the junit test that actually tests that trying to add a dupe will throw an error?